civiclens/nlp/topics.py

```diff
@@ -85,30 +73,25 @@ def run_model(self, docs: list[str]):
 
         try:
             numeric_topics, probs = self.model.fit_transform(input)
-        except (ValueError, TypeError):
-            # log error somewhere
+        except Exception as e:
+            print(f"Hugging Face error: {e}")
             return {}
 
         num_topics = max(numeric_topics)
 
         if num_topics < 0:
```

Comment:
<= ?

Reply:
topics are indexed at 0, however the model sometimes create a -1 topic that reflects outlier, garbage topics.